pkg/chaosdaemon/iptables_server.go

```diff
@@ -41,61 +41,177 @@ func (s *daemonServer) FlushIptables(ctx context.Context, req *pb.IpTablesReques
 	}
 
 	nsPath := GetNsPath(pid, netNS)
-	rule := req.Rule
-
-	format := ""
-	switch rule.Direction {
-	case pb.Rule_INPUT:
-		format = "%s INPUT -m set --match-set %s src -j DROP -w 5"
-	case pb.Rule_OUTPUT:
-		format = "%s OUTPUT -m set --match-set %s dst -j DROP -w 5"
-	default:
-		return nil, fmt.Errorf("unknown rule direction")
-	}
-
-	switch rule.Action {
-	case pb.Rule_ADD:
-		command := fmt.Sprintf(format, "-A", rule.Set)
-		cmd := withNetNS(ctx, nsPath, iptablesCmd, strings.Split(command, " ")...)
-		err = s.addIptablesRules(ctx, cmd)
-	case pb.Rule_DELETE:
-		command := fmt.Sprintf(format, "-D", rule.Set)
-		cmd := withNetNS(ctx, nsPath, iptablesCmd, strings.Split(command, " ")...)
-		err = s.deleteIptablesRules(ctx, cmd)
-	default:
-		return nil, fmt.Errorf("unknown rule action")
+
+	iptables := buildIptablesClient(ctx, nsPath)
+	err = iptables.initializeEnv()
```

Comment:
If the `chains` already exist, what happens?

Reply:
The existing chains will be flushed into an empty chain.